test/unit/specs/components/staking/PageValidator.spec.js

```diff
@@ -262,6 +263,160 @@ describe(`PageValidator`, () => {
   })
 })
 
+describe(`modalOptions`, () => {
+  it(`always shows wallet in the first position`, () => {
+    const $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      getters: getterValues
+    }
+
+    const {
+      vm: { modalOptions }
+    } = mount(PageValidator, {
+      mocks: {
+        $route: {
+          params: { validator: delegate.owner }
+        },
+        $store
+      }
+    })
+
+    const myWallet = {
+      address: $store.getters.wallet.address,
+      maximum: Math.floor(
+        $store.getters.totalAtoms - $store.getters.oldBondedAtoms
+      ),
+      key: `My Wallet - ${shortAddress($store.getters.wallet.address, 20)}`,
```

Comment:
tests should not have too much logic and dependencies imo. I would have just used the actual final string here. This is also better to see the expected outcome. not a blocker for me.

Reply:
addressed